src/lib.rs

```diff
@@ -167,6 +173,9 @@ use search::Search;
 use users::Users;
 
 const DEFAULT_HOST: &str = "https://api.github.com";
+const MAX_JWT_TOKEN_LIFE: time::Duration = time::Duration::from_secs(60 * 10);
+// 9 minutes so we refresh sooner than it actually expires
+const JWT_TOKEN_REFRESH_PERIOD: time::Duration = time::Duration::from_secs(60 * 9);
```

Comment:
After a day in production, I found using 10/9 wasn't adequate. The 10min token would periodically be too far in the future. Not sure if my clock or their clock is drifting -- both I'm sure have NTP. Anyway, I changed to 9/8 and haven't had problems since.